docs/UserGuide.md

```diff
@@ -3,271 +3,478 @@ layout: page
+  
+  ![with policy URL](images/find-then-policy.png)
+
+[Return to Table of Contents](#table-of-contents)
+
 
-### Sorting the list of clients : `sort`
+### Delete client: `delete`
 
-Sorts the list of clients in ClientBook.
+**Purpose**: Deletes the specified client from ClientBook after listing or finding contacts.
 
-Format: `sort ATTRIBUTE DIRECTION`
+**Format**: `delete INDEX`
+
+* Deletes the client at the specified `INDEX`.
+* The index refers to the index number shown in the displayed client list.
+* The index must be more than 1, and less than or equal to the index of the last item in the displayed list.
+
+**Examples**:
+* `list` followed by `delete 2` deletes the 2nd person in the address book.
+
+  ![delete 2](images/delete-2.png)
+
+* `find n/Charlotte` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+
+  ![delete charlotte](images/delete-charlotte.png)
+
+[Return to Table of Contents](#table-of-contents)
+
+
+### Sort list of clients: `sort`
+
+**Purpose**: Sorts the current list of clients in ClientBook.
+
+**Format**: `sort -ATTRIBUTE -DIRECTION`
 
 * Sorts the list of clients according to the specified `ATTRIBUTE` and `DIRECTION`.
-* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy.
-* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order.
+* The specified `ATTRIBUTE` can be `-n` to sort by name or `-i` to sort by insurance policy, but not both.
+* The specified `DIRECTION` can be `-asc` for ascending order or `-des` for descending order, but not both.
 
-Examples:
-* `sort -n -asc` sorts the list of clients by name in ascending order.
+**Examples**:
+* Sort the current list of clients by name in **ascending** order.
+  * `sort -n -asc`
 
-### Exiting the program : `exit`
+    ![sort](images/sort-asc.png)
+
+* Sort the current list of clients by name in **descending** order.
+    * `sort -n -des`
 
-Exits the program.
+      ![sort](images/sort-des.png)
 
-Format: `exit`
+[Return to Table of Contents](#table-of-contents)
 
-### Locking ClientBook : `lock`
 
-Locks ClientBook.
+### Lock ClientBook with a user-selected password: `lock`
 
-Format: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
+**Purpose**: Locks ClientBook with a user-selected password.
+
+**Format**: `lock [CURRENT_PASSWORD] [NEW_PASSWORD]`
 
 * Verifies the current password before locking ClientBook with the new password.
 * The `CURRENT_PASSWORD` field can be omitted if ClientBook is not yet locked.
 * When `CURRENT_PASSWORD` and `NEW_PASSWORD` fields are both omitted, ClientBook will attempt to lock itself using the last used password that is safely stored on your device.
 
-### Unlocking ClientBook : `unlock`
+:exclamation: **Note**: After setting a password, application can only be launched through Command Prompt or Terminal.
```

Comment:
Maybe "After setting a password, **the** application" or **ClientBook** instead of application.